[2025-04-09T08:45:23.731+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T08:45:23.867+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_weather_etl.run_postgres_script manual__2025-04-09T08:44:47.008082+00:00 [queued]>
[2025-04-09T08:45:23.880+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_weather_etl.run_postgres_script manual__2025-04-09T08:44:47.008082+00:00 [queued]>
[2025-04-09T08:45:23.881+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-09T08:45:23.897+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_postgres_script> on 2025-04-09 08:44:47.008082+00:00
[2025-04-09T08:45:23.903+0000] {standard_task_runner.py:72} INFO - Started process 11220 to run task
[2025-04-09T08:45:23.907+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_weather_etl', 'run_postgres_script', 'manual__2025-04-09T08:44:47.008082+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp0ocy4bdg']
[2025-04-09T08:45:23.909+0000] {standard_task_runner.py:105} INFO - Job 258: Subtask run_postgres_script
[2025-04-09T08:45:23.955+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_weather_etl.run_postgres_script manual__2025-04-09T08:44:47.008082+00:00 [running]> on host 3c3c1f66214b
[2025-04-09T08:45:24.026+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_weather_etl' AIRFLOW_CTX_TASK_ID='run_postgres_script' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T08:44:47.008082+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T08:44:47.008082+00:00'
[2025-04-09T08:45:24.028+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-09T08:45:24.029+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-09T08:45:24.031+0000] {logging_mixin.py:190} INFO - Current task name:run_postgres_script state:running start_date:2025-04-09 08:45:23.868783+00:00
[2025-04-09T08:45:24.032+0000] {logging_mixin.py:190} INFO - Dag name:daily_weather_etl and current dag run status:running
[2025-04-09T08:45:24.033+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T08:45:24.034+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-09T08:45:24.035+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/postgres.py']
[2025-04-09T08:45:24.046+0000] {subprocess.py:99} INFO - Output:
[2025-04-09T08:45:27.170+0000] {subprocess.py:106} INFO - 25/04/09 08:45:27 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-04-09T08:45:27.327+0000] {subprocess.py:106} INFO - Setting default log level to "WARN".
[2025-04-09T08:45:27.328+0000] {subprocess.py:106} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-04-09T08:45:33.350+0000] {subprocess.py:106} INFO - 25/04/09 08:45:33 ERROR Executor: Exception in task 0.0 in stage 2.0 (TID 2)
[2025-04-09T08:45:33.351+0000] {subprocess.py:106} INFO - java.sql.BatchUpdateException: Batch entry 0 INSERT INTO pro_raw_api.daily_weather ("city","time","temperature","wind_kph","humidity","feelslike","precip_mm","text") VALUES ('elche','2025-04-07 04:00',15.6,11.2,51,15.6,0.0,'Mostly Clear') was aborted: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.353+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.354+0000] {subprocess.py:106} INFO -   Position: 133  Call getNextException to see other errors in the batch.
[2025-04-09T08:45:33.355+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:171)
[2025-04-09T08:45:33.356+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:862)
[2025-04-09T08:45:33.358+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:901)
[2025-04-09T08:45:33.359+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgPreparedStatement.executeBatch(PgPreparedStatement.java:1644)
[2025-04-09T08:45:33.360+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.savePartition(JdbcUtils.scala:753)
[2025-04-09T08:45:33.361+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1(JdbcUtils.scala:904)
[2025-04-09T08:45:33.362+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1$adapted(JdbcUtils.scala:903)
[2025-04-09T08:45:33.363+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2(RDD.scala:1039)
[2025-04-09T08:45:33.364+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2$adapted(RDD.scala:1039)
[2025-04-09T08:45:33.366+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.$anonfun$runJob$5(SparkContext.scala:2433)
[2025-04-09T08:45:33.366+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-04-09T08:45:33.367+0000] {subprocess.py:106} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-04-09T08:45:33.368+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-04-09T08:45:33.369+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-04-09T08:45:33.371+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-04-09T08:45:33.373+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-04-09T08:45:33.374+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-04-09T08:45:33.376+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-04-09T08:45:33.377+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-04-09T08:45:33.378+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-04-09T08:45:33.379+0000] {subprocess.py:106} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-04-09T08:45:33.381+0000] {subprocess.py:106} INFO - Caused by: org.postgresql.util.PSQLException: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.382+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.383+0000] {subprocess.py:106} INFO -   Position: 133
[2025-04-09T08:45:33.384+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2565)
[2025-04-09T08:45:33.386+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2297)
[2025-04-09T08:45:33.387+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:322)
[2025-04-09T08:45:33.388+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:859)
[2025-04-09T08:45:33.390+0000] {subprocess.py:106} INFO - 	... 19 more
[2025-04-09T08:45:33.391+0000] {subprocess.py:106} INFO - 25/04/09 08:45:33 WARN TaskSetManager: Lost task 0.0 in stage 2.0 (TID 2) (3c3c1f66214b executor driver): java.sql.BatchUpdateException: Batch entry 0 INSERT INTO pro_raw_api.daily_weather ("city","time","temperature","wind_kph","humidity","feelslike","precip_mm","text") VALUES ('elche','2025-04-07 04:00',15.6,11.2,51,15.6,0.0,'Mostly Clear') was aborted: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.392+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.394+0000] {subprocess.py:106} INFO -   Position: 133  Call getNextException to see other errors in the batch.
[2025-04-09T08:45:33.395+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:171)
[2025-04-09T08:45:33.396+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:862)
[2025-04-09T08:45:33.397+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:901)
[2025-04-09T08:45:33.398+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgPreparedStatement.executeBatch(PgPreparedStatement.java:1644)
[2025-04-09T08:45:33.399+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.savePartition(JdbcUtils.scala:753)
[2025-04-09T08:45:33.400+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1(JdbcUtils.scala:904)
[2025-04-09T08:45:33.401+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1$adapted(JdbcUtils.scala:903)
[2025-04-09T08:45:33.402+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2(RDD.scala:1039)
[2025-04-09T08:45:33.403+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2$adapted(RDD.scala:1039)
[2025-04-09T08:45:33.404+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.$anonfun$runJob$5(SparkContext.scala:2433)
[2025-04-09T08:45:33.406+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-04-09T08:45:33.408+0000] {subprocess.py:106} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-04-09T08:45:33.409+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-04-09T08:45:33.410+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-04-09T08:45:33.412+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-04-09T08:45:33.413+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-04-09T08:45:33.414+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-04-09T08:45:33.415+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-04-09T08:45:33.416+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-04-09T08:45:33.417+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-04-09T08:45:33.418+0000] {subprocess.py:106} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-04-09T08:45:33.418+0000] {subprocess.py:106} INFO - Caused by: org.postgresql.util.PSQLException: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.419+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.420+0000] {subprocess.py:106} INFO -   Position: 133
[2025-04-09T08:45:33.421+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2565)
[2025-04-09T08:45:33.423+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2297)
[2025-04-09T08:45:33.424+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:322)
[2025-04-09T08:45:33.425+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:859)
[2025-04-09T08:45:33.427+0000] {subprocess.py:106} INFO - 	... 19 more
[2025-04-09T08:45:33.428+0000] {subprocess.py:106} INFO - 
[2025-04-09T08:45:33.430+0000] {subprocess.py:106} INFO - 25/04/09 08:45:33 ERROR TaskSetManager: Task 0 in stage 2.0 failed 1 times; aborting job
[2025-04-09T08:45:33.642+0000] {subprocess.py:106} INFO - +--------------------+----------------+-----------+--------+--------+---------+---------+------------+
[2025-04-09T08:45:33.643+0000] {subprocess.py:106} INFO - |                city|            time|temperature|wind_kph|humidity|feelslike|precip_mm|        text|
[2025-04-09T08:45:33.644+0000] {subprocess.py:106} INFO - +--------------------+----------------+-----------+--------+--------+---------+---------+------------+
[2025-04-09T08:45:33.645+0000] {subprocess.py:106} INFO - |               elche|2025-04-07 04:00|       15.6|    11.2|      51|     15.6|      0.0|Mostly Clear|
[2025-04-09T08:45:33.646+0000] {subprocess.py:106} INFO - |               elche|2025-04-07 09:00|       16.4|    12.1|      51|     16.4|      0.0|       Sunny|
[2025-04-09T08:45:33.646+0000] {subprocess.py:106} INFO - |               elche|2025-04-07 15:00|       26.2|    12.9|      26|     26.2|      0.0|       Sunny|
[2025-04-09T08:45:33.647+0000] {subprocess.py:106} INFO - |               elche|2025-04-07 22:00|       17.8|     0.8|      54|     17.8|      0.0|       Sunny|
[2025-04-09T08:45:33.648+0000] {subprocess.py:106} INFO - |              madrid|2025-04-07 04:00|        9.0|     2.0|      86|      9.0|      0.0|       Sunny|
[2025-04-09T08:45:33.649+0000] {subprocess.py:106} INFO - |              madrid|2025-04-07 09:00|        8.1|     6.2|      87|      8.1|      0.0|       Sunny|
[2025-04-09T08:45:33.650+0000] {subprocess.py:106} INFO - |              madrid|2025-04-07 15:00|       19.7|     2.2|      46|     19.7|      0.0|       Sunny|
[2025-04-09T08:45:33.651+0000] {subprocess.py:106} INFO - |              madrid|2025-04-07 22:00|       15.0|     4.1|      72|     15.0|      0.0|       Sunny|
[2025-04-09T08:45:33.652+0000] {subprocess.py:106} INFO - |san juan de alicante|2025-04-07 04:00|       13.5|     7.4|      61|     13.5|      0.0|       Sunny|
[2025-04-09T08:45:33.653+0000] {subprocess.py:106} INFO - |san juan de alicante|2025-04-07 09:00|       15.8|     2.7|      53|     15.8|      0.0|       Sunny|
[2025-04-09T08:45:33.654+0000] {subprocess.py:106} INFO - |san juan de alicante|2025-04-07 15:00|       23.7|    11.2|      33|     23.7|      0.0|Mostly Clear|
[2025-04-09T08:45:33.656+0000] {subprocess.py:106} INFO - |san juan de alicante|2025-04-07 22:00|       17.7|     1.5|      57|     17.7|      0.0|       Sunny|
[2025-04-09T08:45:33.658+0000] {subprocess.py:106} INFO - |          torrevieja|2025-04-07 04:00|       15.6|    13.4|      62|     15.6|      0.0|       Sunny|
[2025-04-09T08:45:33.659+0000] {subprocess.py:106} INFO - |          torrevieja|2025-04-07 09:00|       15.6|    15.3|      61|     15.6|      0.0|       Sunny|
[2025-04-09T08:45:33.660+0000] {subprocess.py:106} INFO - |          torrevieja|2025-04-07 15:00|       22.1|    10.0|      53|     22.1|      0.0|       Sunny|
[2025-04-09T08:45:33.661+0000] {subprocess.py:106} INFO - |          torrevieja|2025-04-07 22:00|       15.9|     0.6|      85|     15.9|      0.0|       Sunny|
[2025-04-09T08:45:33.662+0000] {subprocess.py:106} INFO - |            valencia|2025-04-07 04:00|       14.1|     9.6|      70|     14.1|      0.0|       Sunny|
[2025-04-09T08:45:33.663+0000] {subprocess.py:106} INFO - |            valencia|2025-04-07 09:00|       14.1|     8.8|      66|     14.1|      0.0|       Sunny|
[2025-04-09T08:45:33.664+0000] {subprocess.py:106} INFO - |            valencia|2025-04-07 15:00|       22.6|    13.2|      37|     22.6|      0.0|       Sunny|
[2025-04-09T08:45:33.665+0000] {subprocess.py:106} INFO - |            valencia|2025-04-07 22:00|       17.1|     5.4|      69|     17.1|      0.0|       Sunny|
[2025-04-09T08:45:33.666+0000] {subprocess.py:106} INFO - +--------------------+----------------+-----------+--------+--------+---------+---------+------------+
[2025-04-09T08:45:33.667+0000] {subprocess.py:106} INFO - 
[2025-04-09T08:45:33.668+0000] {subprocess.py:106} INFO - Error: An error occurred while calling o43.save.
[2025-04-09T08:45:33.669+0000] {subprocess.py:106} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 2.0 failed 1 times, most recent failure: Lost task 0.0 in stage 2.0 (TID 2) (3c3c1f66214b executor driver): java.sql.BatchUpdateException: Batch entry 0 INSERT INTO pro_raw_api.daily_weather ("city","time","temperature","wind_kph","humidity","feelslike","precip_mm","text") VALUES ('elche','2025-04-07 04:00',15.6,11.2,51,15.6,0.0,'Mostly Clear') was aborted: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.670+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.671+0000] {subprocess.py:106} INFO -   Position: 133  Call getNextException to see other errors in the batch.
[2025-04-09T08:45:33.672+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:171)
[2025-04-09T08:45:33.673+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:862)
[2025-04-09T08:45:33.674+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:901)
[2025-04-09T08:45:33.675+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgPreparedStatement.executeBatch(PgPreparedStatement.java:1644)
[2025-04-09T08:45:33.676+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.savePartition(JdbcUtils.scala:753)
[2025-04-09T08:45:33.678+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1(JdbcUtils.scala:904)
[2025-04-09T08:45:33.679+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1$adapted(JdbcUtils.scala:903)
[2025-04-09T08:45:33.679+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2(RDD.scala:1039)
[2025-04-09T08:45:33.680+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2$adapted(RDD.scala:1039)
[2025-04-09T08:45:33.682+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.$anonfun$runJob$5(SparkContext.scala:2433)
[2025-04-09T08:45:33.683+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-04-09T08:45:33.683+0000] {subprocess.py:106} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-04-09T08:45:33.684+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-04-09T08:45:33.685+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-04-09T08:45:33.688+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-04-09T08:45:33.689+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-04-09T08:45:33.690+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-04-09T08:45:33.691+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-04-09T08:45:33.692+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-04-09T08:45:33.693+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-04-09T08:45:33.694+0000] {subprocess.py:106} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-04-09T08:45:33.695+0000] {subprocess.py:106} INFO - Caused by: org.postgresql.util.PSQLException: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.696+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.697+0000] {subprocess.py:106} INFO -   Position: 133
[2025-04-09T08:45:33.697+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2565)
[2025-04-09T08:45:33.698+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2297)
[2025-04-09T08:45:33.699+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:322)
[2025-04-09T08:45:33.699+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:859)
[2025-04-09T08:45:33.700+0000] {subprocess.py:106} INFO - 	... 19 more
[2025-04-09T08:45:33.701+0000] {subprocess.py:106} INFO - 
[2025-04-09T08:45:33.701+0000] {subprocess.py:106} INFO - Driver stacktrace:
[2025-04-09T08:45:33.702+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2025-04-09T08:45:33.703+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2025-04-09T08:45:33.703+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2025-04-09T08:45:33.704+0000] {subprocess.py:106} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2025-04-09T08:45:33.705+0000] {subprocess.py:106} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2025-04-09T08:45:33.706+0000] {subprocess.py:106} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2025-04-09T08:45:33.707+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2025-04-09T08:45:33.708+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2025-04-09T08:45:33.709+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2025-04-09T08:45:33.709+0000] {subprocess.py:106} INFO - 	at scala.Option.foreach(Option.scala:407)
[2025-04-09T08:45:33.710+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2025-04-09T08:45:33.712+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2025-04-09T08:45:33.712+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2025-04-09T08:45:33.713+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2025-04-09T08:45:33.714+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2025-04-09T08:45:33.714+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2025-04-09T08:45:33.715+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2393)
[2025-04-09T08:45:33.715+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2414)
[2025-04-09T08:45:33.716+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2433)
[2025-04-09T08:45:33.717+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2458)
[2025-04-09T08:45:33.717+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$1(RDD.scala:1039)
[2025-04-09T08:45:33.718+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
[2025-04-09T08:45:33.718+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
[2025-04-09T08:45:33.719+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.withScope(RDD.scala:410)
[2025-04-09T08:45:33.720+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.foreachPartition(RDD.scala:1037)
[2025-04-09T08:45:33.720+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.Dataset.$anonfun$foreachPartition$1(Dataset.scala:3516)
[2025-04-09T08:45:33.721+0000] {subprocess.py:106} INFO - 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[2025-04-09T08:45:33.723+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.Dataset.$anonfun$withNewRDDExecutionId$1(Dataset.scala:4310)
[2025-04-09T08:45:33.724+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2025-04-09T08:45:33.725+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2025-04-09T08:45:33.725+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2025-04-09T08:45:33.726+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2025-04-09T08:45:33.727+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2025-04-09T08:45:33.728+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.Dataset.withNewRDDExecutionId(Dataset.scala:4308)
[2025-04-09T08:45:33.728+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.Dataset.foreachPartition(Dataset.scala:3516)
[2025-04-09T08:45:33.729+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.saveTable(JdbcUtils.scala:903)
[2025-04-09T08:45:33.730+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcRelationProvider.createRelation(JdbcRelationProvider.scala:70)
[2025-04-09T08:45:33.730+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.SaveIntoDataSourceCommand.run(SaveIntoDataSourceCommand.scala:48)
[2025-04-09T08:45:33.731+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.command.ExecutedCommandExec.sideEffectResult$lzycompute(commands.scala:75)
[2025-04-09T08:45:33.731+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.command.ExecutedCommandExec.sideEffectResult(commands.scala:73)
[2025-04-09T08:45:33.732+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.command.ExecutedCommandExec.executeCollect(commands.scala:84)
[2025-04-09T08:45:33.733+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.$anonfun$applyOrElse$1(QueryExecution.scala:107)
[2025-04-09T08:45:33.734+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2025-04-09T08:45:33.734+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2025-04-09T08:45:33.735+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2025-04-09T08:45:33.735+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2025-04-09T08:45:33.736+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2025-04-09T08:45:33.737+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:107)
[2025-04-09T08:45:33.737+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution$$anonfun$eagerlyExecuteCommands$1.applyOrElse(QueryExecution.scala:98)
[2025-04-09T08:45:33.738+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.$anonfun$transformDownWithPruning$1(TreeNode.scala:461)
[2025-04-09T08:45:33.739+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.trees.CurrentOrigin$.withOrigin(origin.scala:76)
[2025-04-09T08:45:33.740+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDownWithPruning(TreeNode.scala:461)
[2025-04-09T08:45:33.741+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.org$apache$spark$sql$catalyst$plans$logical$AnalysisHelper$$super$transformDownWithPruning(LogicalPlan.scala:32)
[2025-04-09T08:45:33.742+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning(AnalysisHelper.scala:267)
[2025-04-09T08:45:33.742+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.AnalysisHelper.transformDownWithPruning$(AnalysisHelper.scala:263)
[2025-04-09T08:45:33.743+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2025-04-09T08:45:33.743+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.plans.logical.LogicalPlan.transformDownWithPruning(LogicalPlan.scala:32)
[2025-04-09T08:45:33.744+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.catalyst.trees.TreeNode.transformDown(TreeNode.scala:437)
[2025-04-09T08:45:33.745+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution.eagerlyExecuteCommands(QueryExecution.scala:98)
[2025-04-09T08:45:33.745+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted$lzycompute(QueryExecution.scala:85)
[2025-04-09T08:45:33.746+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution.commandExecuted(QueryExecution.scala:83)
[2025-04-09T08:45:33.747+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.QueryExecution.assertCommandExecuted(QueryExecution.scala:142)
[2025-04-09T08:45:33.747+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.DataFrameWriter.runCommand(DataFrameWriter.scala:869)
[2025-04-09T08:45:33.748+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveToV1Source(DataFrameWriter.scala:391)
[2025-04-09T08:45:33.749+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.DataFrameWriter.saveInternal(DataFrameWriter.scala:364)
[2025-04-09T08:45:33.749+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.DataFrameWriter.save(DataFrameWriter.scala:251)
[2025-04-09T08:45:33.750+0000] {subprocess.py:106} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2025-04-09T08:45:33.750+0000] {subprocess.py:106} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2025-04-09T08:45:33.751+0000] {subprocess.py:106} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2025-04-09T08:45:33.752+0000] {subprocess.py:106} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2025-04-09T08:45:33.752+0000] {subprocess.py:106} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2025-04-09T08:45:33.753+0000] {subprocess.py:106} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-04-09T08:45:33.754+0000] {subprocess.py:106} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2025-04-09T08:45:33.755+0000] {subprocess.py:106} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2025-04-09T08:45:33.756+0000] {subprocess.py:106} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2025-04-09T08:45:33.757+0000] {subprocess.py:106} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-04-09T08:45:33.757+0000] {subprocess.py:106} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-04-09T08:45:33.758+0000] {subprocess.py:106} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-04-09T08:45:33.759+0000] {subprocess.py:106} INFO - Caused by: java.sql.BatchUpdateException: Batch entry 0 INSERT INTO pro_raw_api.daily_weather ("city","time","temperature","wind_kph","humidity","feelslike","precip_mm","text") VALUES ('elche','2025-04-07 04:00',15.6,11.2,51,15.6,0.0,'Mostly Clear') was aborted: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.759+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.760+0000] {subprocess.py:106} INFO -   Position: 133  Call getNextException to see other errors in the batch.
[2025-04-09T08:45:33.761+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:171)
[2025-04-09T08:45:33.761+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:862)
[2025-04-09T08:45:33.762+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:901)
[2025-04-09T08:45:33.763+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgPreparedStatement.executeBatch(PgPreparedStatement.java:1644)
[2025-04-09T08:45:33.763+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.savePartition(JdbcUtils.scala:753)
[2025-04-09T08:45:33.764+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1(JdbcUtils.scala:904)
[2025-04-09T08:45:33.764+0000] {subprocess.py:106} INFO - 	at org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils$.$anonfun$saveTable$1$adapted(JdbcUtils.scala:903)
[2025-04-09T08:45:33.765+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2(RDD.scala:1039)
[2025-04-09T08:45:33.766+0000] {subprocess.py:106} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$foreachPartition$2$adapted(RDD.scala:1039)
[2025-04-09T08:45:33.766+0000] {subprocess.py:106} INFO - 	at org.apache.spark.SparkContext.$anonfun$runJob$5(SparkContext.scala:2433)
[2025-04-09T08:45:33.767+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-04-09T08:45:33.767+0000] {subprocess.py:106} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-04-09T08:45:33.768+0000] {subprocess.py:106} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-04-09T08:45:33.769+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-04-09T08:45:33.769+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-04-09T08:45:33.770+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-04-09T08:45:33.771+0000] {subprocess.py:106} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-04-09T08:45:33.773+0000] {subprocess.py:106} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-04-09T08:45:33.774+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-04-09T08:45:33.774+0000] {subprocess.py:106} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-04-09T08:45:33.775+0000] {subprocess.py:106} INFO - 	... 1 more
[2025-04-09T08:45:33.776+0000] {subprocess.py:106} INFO - Caused by: org.postgresql.util.PSQLException: ERROR: column "time" is of type timestamp without time zone but expression is of type character varying
[2025-04-09T08:45:33.776+0000] {subprocess.py:106} INFO -   Hint: You will need to rewrite or cast the expression.
[2025-04-09T08:45:33.777+0000] {subprocess.py:106} INFO -   Position: 133
[2025-04-09T08:45:33.778+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2565)
[2025-04-09T08:45:33.778+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2297)
[2025-04-09T08:45:33.779+0000] {subprocess.py:106} INFO - 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:322)
[2025-04-09T08:45:33.780+0000] {subprocess.py:106} INFO - 	at org.postgresql.jdbc.PgStatement.internalExecuteBatch(PgStatement.java:859)
[2025-04-09T08:45:33.780+0000] {subprocess.py:106} INFO - 	... 19 more
[2025-04-09T08:45:34.030+0000] {subprocess.py:106} INFO - 
[2025-04-09T08:45:34.525+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-09T08:45:34.543+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T08:45:34.545+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=daily_weather_etl, task_id=run_postgres_script, run_id=manual__2025-04-09T08:44:47.008082+00:00, execution_date=20250409T084447, start_date=20250409T084523, end_date=20250409T084534
[2025-04-09T08:45:34.563+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-09T08:45:34.565+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-09T08:45:34.565+0000] {logging_mixin.py:190} INFO - Dag name:daily_weather_etl queued_at:2025-04-09 08:44:47.022169+00:00
[2025-04-09T08:45:34.566+0000] {logging_mixin.py:190} INFO - Task hostname:3c3c1f66214b operator:BashOperator
[2025-04-09T08:45:34.577+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T08:45:34.590+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T08:45:34.593+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
